.env.example

```diff
@@ -3,7 +3,7 @@ PORT=8080
 # Permitted domains for cross-origin requests, e.g. http://localhost:1313
 ALLOWED_ORIGINS=
 # Filepath to domain blacklist
-DOMAIN_BLACKLIST=
+DOMAIN_BLACKLIST=domain_blacklist.txt
```

Comment:
Good point! I think it we specify a default in `.env.example` we won't be able to differentiate between a real fail and a fail because the user doesn't want a domain blacklist. I updated `.env.example` to just suggest that (gitignored) filepath in a comment.